imap_processing/swe/l1a/swe_l1a.py

```diff
@@ -13,15 +21,51 @@ def swe_l1a(packet_file: str):
 
     Parameters
     ----------
-    packet_file : str
-        The path and filename to the L0 file to read
+    packets: list
+        Decom data list that contains all appIds
+
+    Returns
+    -------
+    str
+        Path name of where CDF file was created.
+        This is used to upload file from local to s3.
+        TODO: test this later.
     """
-    decom_data = decom_swe.decom_packets(packet_file)
-    logging.info(f"Unpacking data from {packet_file}")
-
-    # If appId is science, then the file should contain all data of science appId
-    if decom_data[0].header["PKT_APID"].raw_value == 1344:
-        logging.info("Processing science data")
-        return swe_science(decom_data=decom_data)
-    else:
-        return decom_data
+    # group data by appId
+    grouped_data = group_by_apid(packets)
+
+    for apid in grouped_data.keys():
+        # If appId is science, then the file should contain all data of science appId
+        if apid == SWEAPID.SWE_SCIENCE.value:
```

Comment:
```suggestion
        if apid == SWEAPID.SWE_SCIENCE:
```
since you're using an IntEnum I don't think you need the value here and below.

Reply:
interesting. I didn't know that.